packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -234,7 +234,9 @@ export default function MainFeedLayout({
     <LayoutHeader className="flex-col">
       {alerts?.filter && (
         <CreateMyFeedButton
-          action={() => onInitializeOnboarding(onFeedPageChanged)}
+          action={() =>
+            onInitializeOnboarding(() => onFeedPageChanged(MainFeedPage.MyFeed))
+          }
```

Comment:
Why did we need to change something here?
Noting this so we can make sure to test it properly.